protocols/ping/src/handler.rs

```diff
@@ -225,6 +221,7 @@ impl ConnectionHandler for Handler {
 
     fn on_behaviour_event(&mut self, _: Void) {}
 
+    #[tracing::instrument(level = "trace", name = "ConnectionHandler::poll", skip(self, cx))]
```

Comment:
Before we merge this, we need to apply this to all handlers of all protocols. Same for `NetworkBehaviour`.